js/main.js

```diff
@@ -527,3 +527,139 @@ var xint = setInterval(function() {
   addEvent(document.body, 'mousedown', makeEditable);
   clearInterval(xint);
 }, 200);
+
+function generateDonationUrl(address, amountBtc, message) {
+    var result = [
+        address
+    ];
+
+    amountBtc = parseFloat(amountBtc);
+
+    if (!isNaN(amountBtc)) {
```

Comment:
I missed the `parseFloat()` statement, sorry about that.
The greater than `0` checks is useful in protecting users from themselves. :)